index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar" class="nav_menu">
+      <ul class="nav-items">
+        <li><a href="#welcome-section"> About me </a></li>
+        <li><a href="#projects"> My Projects </a></li>
+        <li><a href="#contact"> Contact me </a></li>
+      </ul>
+    </nav>
+
+    <section id="welcome-section" class="intro-sect">
+      <h1>Hi I am Morrighan</h1>
+      <p>a web developer</p>
+    </section>
+
+    <section id="projects" class="pr_section">
+      <h2>Some examples of projects</h2>
+
+      <div class="pr_grid">
+        <a href="#" target="_blank" class="project_tile">
+          <img class="pr-img" src="img/hangman.jpg" alt="hangman game" />
+          <p class="pr_name">Hangman Game</p>
+        </a>
```

Comment:
Please consider using a semantic HTML tag like `<article>`, it would make the code more clear.